core/src/core/temporal/utils/execution.py

```
@@ -0,0 +1,219 @@
+        start_to_close_timeout=timedelta(
+            seconds=resilience_config.execution_timeout
+            or pipeline_config.default_timeout
+        ),
+        retry_policy=create_retry_policy(resilience_config),
+    )
+
+
+async def _execute_tasks_with_error_handling(
+    tasks: list[tuple[str, Any]],
+) -> list[StageExecutionResult]:
+    """Выполняет задачи с обработкой ошибок."""
```

Ruff (0.12.2)

Docstring contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF002)